merge-eslint-reports.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const directories = fs.readdirSync('./packages');
+const reports = directories.map((dir) => fs.readFileSync(`./packages/${dir}/report.json`, { encoding: 'utf-8' }));
```

Comment:
nit: we can use async read file method and wrapped in a Promise.all to await all files loaded

Reply:
the outmost code block cannot use async

Reply:
Ah, yes... you're right